src/index.ts

```diff
@@ -258,6 +259,29 @@ async function main() {
         await addNewPayment(argv.r, argv.p, argv.a, argv.x);
       }
     )
+    .command(
+      "setmaintenancemode",
+      'Set maintenance mode\n Example: node dist/index.js setmaintenancemode -r "http://127.0.0.1:8080" -u "YOUR_ADDRESS" --active <true|false>',
+      // options for your command
+      function (yargs: any) {
+        return yargs
+          .option("u", {
```

Comment:
This one should be priv key, not address.
Address is  public which means anyone can set our server to maintainance mode